From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sun, 12 Jun 2011 21:57:01 +0200
Subject: Skip nodeinfo test on non intel architectures

since the testfiles assume a /proc/cpuinfo specific to this
architecture.
---
 tests/nodeinfotest.c |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/tests/nodeinfotest.c b/tests/nodeinfotest.c
index b4e81b3..71e2926 100644
--- a/tests/nodeinfotest.c
+++ b/tests/nodeinfotest.c
@@ -11,7 +11,9 @@
 #include "util.h"
 #include "files.h"
 
-#ifndef __linux__
+#if ! (defined __linux__  &&  (defined(__x86_64__) || \
+                               defined(__amd64__)  || \
+                               defined(__i386__)))
 
 static int
 mymain(int argc ATTRIBUTE_UNUSED, char **argv ATTRIBUTE_UNUSED)
-- 
